docs/Manpage.md

```diff
@@ -71,39 +71,9 @@ Homebrew Cask provides a friendly CLI workflow for the administration of macOS a
 
 Commands:
 
-- `audit`
-  <br>Check *`cask`* for Homebrew coding style violations.
-
-- `cat`
-  <br>Dump raw source of a *`cask`* to the standard output.
-
-- `create`
-  <br>Creates the given *`cask`* and opens it in an editor.
-
-- `edit`
-  <br>Open the given *`cask`* for editing.
-
-- `fetch`
-  <br>Downloads remote application files to local cache.
-
 - `help`
   <br>Print help for `cask` commands.
```

Comment:
@reitermarkus Can this get deprecated too? If so, I think we can remove this entire section